feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/domain/interfaces/CoinPriceRepository.kt

```diff
@@ -7,6 +7,10 @@ import retrofit2.HttpException
 import kotlin.jvm.Throws
 import kotlin.time.Duration
 
+enum class Range {
```

Comment:
`Range` sounds a bit generic. I suggest to rename it to somthing like `PriceChartPeriod`. And I think this enum should be in a separate file in `domain/model`